fix_clone.cjs

```
@@ -0,0 +1,64 @@
+const fs = require('fs');
```

ESLint

[error] 'require' is not defined.

(no-undef)